pallets/tickets/src/tests.rs

```diff
@@ -1 +1,600 @@
-// TODO
+use crate::mock;
+use crate as pallet_tickets;
+
+use crate::*;
+
+use crate::dummy_data::*;
+
+use crate::mock::*;
+
+use serde_json::json;
+
+use frame_system::offchain::{
+    SignedPayload, SigningTypes,
+};
+
+use frame_support::{
+    assert_noop, assert_ok,
+    traits::{
+        TypedGet,
+    }
+};
+
+use frame_support::dispatch::DispatchResult;
+
+use sp_core::{
+
+    H160,
+
```

Comment:
lots of empty spaces in this file that can be removed.

Reply:
👍👍👍